src/transformers/models/vit/modeling_vit.py

```diff
@@ -184,6 +184,36 @@ def forward(self, pixel_values: torch.Tensor, interpolate_pos_encoding: bool = F
         return embeddings
 
 
+def eager_attention_forward(
```

Comment:
missing copied from as well no?

Reply:
It's added by modular, no need for `# Copied from`